src/ansys/aedt/core/modeler/pcb/object_3d_layout.py

```diff
@@ -2301,3 +2359,34 @@ def remove(self):
 
         """
         self.padstackmgr.Remove(self.name, True, "", "Project")
+
+
+def _get_die_properties(model_info):
+    s = r".+DieProp\(dt=(.+?), do=(.+?), dh='(.+?)', lid=(.+?)\)"
+    m = re.search(s, model_info)
+    dt = 0
+    do = 0
+    dh = "0"
+    lid = -100
+    if m:
+        dt = int(m.group(1))
+        do = int(m.group(2))
+        dh = str(m.group(3))
+        lid = int(m.group(4))
+    return dt, do, dh, lid
+
+
+def _get_port_properties(model_info):
+    s = r".+PortProp\(rh='(.+?)', rsa=(.+?), rsx='(.+?)', rsy='(.+?)'\)"
+    m = re.search(s, model_info)
+    rsx = "0"
+    rsy = "0"
+    rsa = True
+    rh = "0"
+    if m:
+        rh = m.group(1)
+        rsx = m.group(3)
+        rsy = m.group(4)
+        if m.group(2) == "false":
+            rsa = False
+    return rh, rsa, rsx, rsy
```

Comment:
Following previous comment, could you make these methods static methods of the class  and make them private if possible ?

Reply:
Done.